docs/pages/reference/config.mdx

```diff
@@ -770,12 +770,18 @@ app_service:
       - name: "os"
         command: ["/usr/bin/uname"]
         period: "5s"
-      # Optional simple rewriting of Location header
-      # Rewrite the "Location" header on redirect responses replacing the
-      # host with the public address of this application.
-      # redirect:
-      #   - "localhost"
-      #   - "jenkins.internal.dev"
+      # Optional list of rewrite rules to apply to requests and responses
+      # rewrite:
+        # Optional simple rewriting of Location header
+        # Rewrite the "Location" header on redirect responses replacing the
+        # host with the public address of this application.
+        # redirect:
+        #   - "localhost"
+        #   - "jenkins.internal.dev"
+        # Optional list of extra headers to inject in to requests.
+        # headers:
+        #   For example:
+        #   - "Host: jenkins.example.com"
```

Comment:
@r0mant as a solution to the [comment above](https://github.com/gravitational/teleport/pull/17496/files#r997423103), how about using two `#`s to differentiant comments versus commented example code?

```suggestion
      ## Optional list of rewrite rules to apply to requests and responses
      # rewrite:
        ## Optional simple rewriting of Location header
        ## Rewrite the "Location" header on redirect responses replacing the
        ## host with the public address of this application.
        # redirect:
        #   - "localhost"
        #   - "jenkins.internal.dev"
        ## Optional list of extra headers to inject in to requests.
        # headers:
        #   For example:
        #   - "Host: jenkins.example.com"
```

Reply:
That's fine.